card/lib/card/cache.rb

```diff
@@ -25,7 +25,7 @@ class Cache
     class << self
       def [] klass
         raise "nil klass" if klass.nil?
-        cache_by_class[klass] ||= new :class=>klass, :store=>(@@using_rails_cache ? nil : Rails.cache)
+        cache_by_class[klass] ||= new :class=>klass, :store=>(@@using_rails_cache ? nil : Cardio.cache)
```

Comment:
Not sure, but you may have changed this to Rails in the merge.  Was that right?  We are not supposed to reference Rails, Wagn or Decko in Card, right?  Let me know if you have a different requirement on that.